install_config/aggregate_logging.adoc

```diff
@@ -1011,6 +1012,121 @@ $ for dc in $(oc get deploymentconfig --selector logging-infra=elasticsearch -o
   done
 ----
 
+[[aggregated-logging-hostPath-Elasticsearch]]
+===== Configuring hostPath storage for Elasticsearch =====
+
+You can provision {product-title} clusters using hostPath storage for Elasticsearch.
+
+To use a local disk volume on each node host as storage for an Elasticsearch replica:
+
+. Create a local mount point on each infrastructure node for the local Elasticsearch storage:
++
+----
+mkdir /usr/local/es-storage
+----
+
+. Create a filesystem on the Elasticsearch volume:
++
+----
+mkfs.ext4 /dev/xxx
+----
+
+. Mount the elasticsearch volume:
++
+----
+mount /dev/xxx /usr/local/es-storage
+----
+
+. Add the following line to `/etc/fstab`:
++
+----
+/dev/xxx /usr/local/es-storage ext4
+----
+
+. Change ownership for the mount point:
++
+----
+chown 1000:1000 /usr/local/es-storage
+----
+
+. Give the privilege to mount and edit a local volume to the relevant service account:
++
+----
+  $ oc adm policy add-scc-to-user privileged  \
+         system:serviceaccount:logging:aggregated-logging-elasticsearch
+----
+  Use the project you created earlier (for example, logging) when running the logging playbook.
```

Comment:
This is appearing as part of a new source block, likely due to the indentation. If it's meant to be part of the previous step in the procedure make it a separate line and include a + before it.  For instance, lines 1058-1059 would become:

    +
    Use the project you created earlier...

Reply:
Fixed so it has correct indentation, etc.. Thanks.